proposals/lip-0063.md

```diff
@@ -218,50 +218,72 @@ ffffff
-[DPoS Module](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0057.md#notation-and-constants)            | `ROUND_LENGTH`                 | 103
-[DPoS Module](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0057.md#notation-and-constants)            | `BFT_THRESHOLD`                | 68
-[DPoS Module](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0057.md#notation-and-constants)            | `MIN_WEIGHT_STANDBY`           | 1000 x (10^8)
-[DPoS Module](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0057.md#notation-and-constants)            | `NUMBER_ACTIVE_DELEGATES`      | 101
-[DPoS Module](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0057.md#notation-and-constants)            | `NUMBER_STANDBY_DELEGATES`     | 2
-[DPoS Module](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0057.md#notation-and-constants)            | `TOKEN_ID_DPOS`                | `TOKEN_ID_LSK = 0x 00 00 00 00 00 00 00 00`
-[DPoS Module](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0057.md#notation-and-constants)            | `DELEGATE_REGISTRATION_FEE`    | 10 x (10^8)
-[Random Module](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0046.md#constants-and-config-parameters) | `MAX_LENGTH_VALIDATOR_REVEALS` | 206
-[Fee Module](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0048.md#notation-and-constants)             | `MIN_FEE_PER_BYTE`             | 1000
-[Fee Module](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0048.md#notation-and-constants)             | `TOKEN_ID_FEE`                 | `TOKEN_ID_LSK = 0x 00 00 00 00 00 00 00 00`
+**Module/LIP**                                                                                                  | **Constant Name**                   | **Value**
+--------------------------------------------------------------------------------------------------------------- | ----------------------------------- | -----------------------------------------------------------------------------------------------------------------------------------------------
+[LIP 0058](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0058.md#constants)                            | `LSK_BFT_BATCH_SIZE`                | 103
+[LIP 0055](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0055.md#notation-and-constants)               | `MAX_TRANSACTIONS_SIZE_BYTES`       | 15,360
+[LIP 0055](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0055.md#notation-and-constants)               | `MAX_ASSET_DATA_SIZE_BYTES`         | 18 (The Random Module is the only module adding an entry to the assets property whereby the size of the data property will not exceed 18 bytes)
+[LIP 0055](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0055.md#notation-and-constants)               | `CHAIN_ID`                          | `0x 00 00 00 00`
```

Comment:
This constant is special in that it would be different for the Betanet, Testnet and Mainnet mainchain. All other constants would be the same for the different networks as far as I see.

Reply:
yes, but I would leave this point open until [this discussion](https://github.com/LiskHQ/discussions/discussions/63) is solved. As this issue is spread over many LIPs, I would also prefer to fix it in a separate PR in which we fix all LIPs at the same time.

Reply:
Okay, can be resolved then.